packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -122,6 +122,19 @@ Optional steps can be added to the wizard. Make steps optional when they do not
 * The review step is enabled once all required steps have been filled out.
 
 * Once required steps have been filled, a tertiary button would appear to the right of the next/back buttons. Once clicked, this button would jump users ahead to the final review step. 
+ 
+### Wizard with a panel
+
+The drawer inside a wizard is useful in situations where you need to show more information while keeping the user in the context of the wizard. When opened, the drawer overlays the data, rather than pushing it to the side. There are two types of drawers: dismissable and non-dismissible. For opening and closing a drawer, use a link button or a link button with an icon.
+
+<img src="./img/wizard-with-panel-closed.png" alt="Wizard with a closed panel"/>
+<img src="./img/wizard-with-panel-open.png" alt="Wizard with a closed open"/>
+
+#### When to use
+
```

Comment:
would add back the "Use drawer when:"

Reply:
would it be "use drawer when" or "use panel when", and should i include the ":" ?

Reply:
You know this makes me realize that it should be wizard with drawer everywhere! So "Use drawer when:", and include the : yes.

Make sure you change the section heading too, and any other mention of "panel" to "drawer"